keyboards/studiokestra/galatea/rev1/config.h

```diff
@@ -0,0 +1,8 @@
+// Copyright 2023 studiokestra (@studiokestra)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define LED_CAPS_LOCK_PIN B5
+#define LED_SCROLL_LOCK_PIN D2
+#define LED_PIN_ON_STATE 0
```

Comment:
these can be moved to the info.json config, as well.

Reply:
Thanks for the feedback @drashna . I've added it to the respective info.json now.